django-stubs/db/models/constraints.pyi

```diff
@@ -27,9 +28,15 @@ class CheckConstraint(BaseConstraint):
     def __init__(self, *, check: Q, name: str) -> None: ...
 
 class UniqueConstraint(BaseConstraint):
+    expressions: Tuple[Combinable]
```

Comment:
`Tuple[Combinable, ...]`?

Reply:
Yes, I think so, I wasn't aware of this syntax, thanks.

I assume this should also be changed for `fields`? I can do this here, but it's not completely relevant to this change.